api/middleware/notesMiddleware.js

```diff
@@ -18,6 +19,48 @@ const checkNoteExists = async (req, res, next) => {
   }
 };
 
+const helper_checkProfileIdExists = async (profile_id) => {
+  const profile = await Profile_Model.findById(profile_id);
+  return Boolean(profile);
+};
+
+const checkMenteeIdExists = async (req, res, next) => {
+  try {
+    // console.log('req.params.mentee_id = ', req.params);
```

Comment:
Why is there a commented console log? I think it can be deleted.

Reply:
i did not catch that comment. Line 29 need deletion.

Reply:
line 29 was deleted